src/NLog/Targets/Target.cs

```diff
@@ -669,6 +669,9 @@ protected void MergeEventProperties(LogEventInfo logEvent)
         /// <returns>String representing log event.</returns>
         protected string RenderLogEvent(Layout layout, LogEventInfo logEvent)
         {
+            if (layout == null || logEvent == null)
+                return string.Empty;
```

Comment:
I think it would be nice to write a warning (maybe debug level) to the internal logger in this case, what do you think?

Reply:
Not sure I like messages per logevent.

Reply:
true, but not logging anything without any logs is also not that nice. Internal log Trace level otherwise?

Reply:
Well some Layouts are expected to be blank/null, so I think the important layout logging should be handled by the individual targets.

Reply:
OK, thanks for the input